assembly/src/event/context.rs

```diff
@@ -71,6 +71,7 @@ impl EventContext<'_> {
     where
         T: VromValueT,
     {
+        self.trace.vrom().check_alignment::<T>(addr)?;
```

Comment:
Unrelated to this PR, but this extra check catches some issues before entering the prover.

Reply:
Yeah this got "removed" inadvertently when fixing the VROM writes some time ago I believe, good catch

Reply:
Yeah this got "removed" inadvertently when fixing the VROM writes some time ago I believe, good catch